zeroex/orderwatch/order_watcher.go

```diff
@@ -826,7 +878,11 @@ func (w *Watcher) unwatchOrder(u updatableOrdersCol, order *meshdb.Order, newFil
 	w.expirationWatcher.Remove(expirationTimestamp, order.Hash.Hex())
 }
 
-func (w *Watcher) permanentlyDeleteOrder(ordersColTxn *db.Transaction, order *meshdb.Order) error {
+type deletableOrdersCol interface {
```

Comment:
I think the idea behind using this `deletableOrdersCol` was to make it compatible with both transactions and the Orders collection? I don't think we ever actually pass in a transaction so maybe we can just remove this interface for now?

Reply:
We are using a transaction when it's called in `generateOrderEventsIfChanged`. I think we need to keep it.

Reply:
Ah I see. That doesn't show up in the diff for this PR.

In that case can we rename this interface to `orderDeleter` to conform with naming conventions for small interfaces in Go and also rename the argument name in `permanentlyDeleteOrder` to reflect this?